sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/implementation/EventHubReactorConnectionTest.java

```diff
@@ -158,6 +160,15 @@ public void getsManagementChannel() {
         StepVerifier.create(connection.getManagementNode())
             .assertNext(node -> Assertions.assertTrue(node instanceof ManagementChannel))
             .verifyComplete();
+
+        System.out.println("-----------------" + connection.createSendLink("send-name", "test-entity-path", new AmqpRetryOptions()));
+
+        StepVerifier.create(connection.createSendLink("send-name", "test-entity-path", new AmqpRetryOptions()))
+            .verifyError(NullPointerException.class);
+
+        StepVerifier.create(connection.createReceiveLink("receiver-name", "test-entity-path", EventPosition.latest(), new ReceiveOptions()))
+            .verifyError(NullPointerException.class);
```

Comment:
I don't think the mocks are set up correctly for testing `createSendLink` and `createReceiveLink` - `NullPointerException` is not the expected behavior here. Also, these have to be in separate test methods as this method is for testing management channel. Send and receive should be in its own test methods.